exercises/flatten-array/flatten_array_test.exs

```diff
@@ -0,0 +1,34 @@
+if !System.get_env("EXERCISM_TEST_EXAMPLES") do
+  Code.load_file("flatten_array.exs")
+end
+
+ExUnit.start
+ExUnit.configure exclude: :pending, trace: true
+
+defmodule ChangeTest do
+  use ExUnit.Case
+
+  test "returns original list if there is nothing to flatten" do
+    assert Flattener.flatten([1, 2, 3]) ==  [1, 2, 3]
+  end
+
+  @tag :pending
+  test "flattens an empty nested list" do
+    assert Flattener.flatten([[]]) ==  []
+  end
+
+  @tag :pending
+  test "flattens a not empty nested list" do
+    assert Flattener.flatten([1,[2,[3],4],5,[6,[7,8]]]) == [1, 2, 3, 4, 5, 6, 7, 8]
+  end
+
+  @tag :pending
+  test "removes nil from list" do
+    assert Flattener.flatten([1, nil, 2]) ==  [1, 2]
+  end
+
+  @tag :pending
```

Comment:
I was thinking something like `[1,[2,nil,4],5]`. It might be helpful for folks to have something between `[1, nil, 2]` and the next test where everything is nil all the way down.